schemas/common/all-types.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "http://jsonlogic.com/schemas/common/all-types.json",
+
+  "title": "All",
+  "description": "Any valid JSON data type.",
+  "type": ["array", "boolean", "null", "number", "object", "string"]
```

Comment:
We need  this for hardcoded static values...
But it forces us to use [`anyOf`](https://json-schema.org/understanding-json-schema/reference/combining.html#anyof) instead of [`oneOf`](https://json-schema.org/understanding-json-schema/reference/combining.html#oneof) 🤔